docs/reference/upgrade.asciidoc

```diff
@@ -47,12 +47,15 @@ a|
 
 [WARNING]
 ====
-The following upgrade paths are *not* supported:
+The following upgrade path is *not* supported (both full cluster restart and rolling upgrade):
 
 * 6.8 to 7.0.
-* 6.7 to 7.1.–{version}.
 
-In-place downgrades to earlier versions are not supported. To downgrade to an
+The following upgrade path is supported but requires a <<restart-upgrade, Full cluster restart upgrade>>.
+
+* 6.0–6.7 directly to {version}.
+
+In-place downgrades to earlier versions are *not supported*. To downgrade to an
```

Comment:
Suggest making this a separate `WARNING` -- it's unrelated to the first bit.

Reply:
I've added a separate warning.